binder/environment.yml

```diff
@@ -1,10 +1,13 @@
 channels:
 - conda-forge
 dependencies:
-- pyiron =0.3.9
-- lammps
-- nglview
-- seaborn
-- sqsgenerator
-- phonopy
-- sphinxdft
+- pyiron =0.4.5
+- pyiron_base=0.3.9
```

Comment:
I guess we can remove `pyiron` and only use `pyiron_atomistics=0.2.30` and `pyiron_base=0.4.2`.

Reply:
I tried the notebooks, but by runing pyiron tables, I faced an error. I reported in https://github.com/pyiron/pyiron_base/issues/559.

Reply:
@jan-janssen, now the latest pyiron_atomistics has replaced pyiron in the environment file. I have tested the binder link, the solution notebooks work fine as well. If you agree, I can merge this.